cms/static/cms/js/require-config.js

```diff
@@ -241,14 +244,26 @@
                 exports: "XBlock",
                 deps: ["xblock/core"]
             },
-
             "coffee/src/main": {
                 deps: ["coffee/src/ajax_prefix"]
             },
             "js/src/logger": {
                 exports: "Logger",
                 deps: ["coffee/src/ajax_prefix"]
             },
+            "modernizr": {
+                exports: "Modernizr"
+            },
+            "afontgarde": {
+                exports: "AFontGarde"
+            },
+            "edxicons": {
```

Comment:
EdxIcons is also RequireJS aware. You can remove this whole section about edxicons.

https://github.com/edx/ux-pattern-library/blob/master/pattern-library/js/edx-icons.js